features/0793-unqualfied-dids-transition/README.md

```diff
@@ -0,0 +1,102 @@
+# Aries RFC 0793: Unqualified DID Transition
+
+- Authors: [Sam Curren](mailto:swcurran@cloudcompass.ca)
+- Status: [PROPOSED](/README.md#proposed)
+- Since: 2023-07-11
+- Status Note: In Pre-work **Target Completion Date: 2023.10.15** 
+- Supersedes:
+- Start Date: 2023-07-11
+- Tags: [feature](/tags.md#feature), [community-update](/tags.md#community-update)
+
+## Summary
+
+Historically, Aries use of the Indy SDK's wallet included the use of 'unqualified DIDs' or DIDs without a did: prefix and method. 
+This transition documents the process of migrating any such DIDs still in use to fully qualified DIDs.
+
+The transition from unqualified DIDs to peer:did:2 is described here: https://github.com/TimoGlastra/legacy-did-transformation
+
+The use and support for did:peer:3 is also included in this transition.
+
+This process ONLY applies to agent codebases using unqualified DIDs. Any codebases not using unqualified DIDs may ignore this process.
+
+This RFC follows the guidance in [RFC
+0345](../../concepts/0345-community-coordinated-update/README.md) about
+community-coordinated updates to (try to) ensure that independently deployed,
+interoperable agents remain interoperable throughout this transition.
+
+The transition from the unqualified to qualified DIDs will occur in four steps:
+
+- **Pre-work**: where we agree on the transition plan outlined in this RFC.
+  - Finalize did:peer:3 details, including feature discovery support of did:peer:3.
+  - Verify transisiton plan and code.
+- **Step 1**: Agent builders MUST update all agent code bases and deployments to accept DIDs in the old (unqualified) and new (transitioned) forms. During this step, agents MUST continue the use of unqualified DIDs in all cases where they are currently being used.
+  - Each agent builder SHOULD notify the community they have completed Step 1 by submitting a PR to update their entry in the [implementations](#implementations) accordingly.
+- **Step 2**: Agent builders MUST update all agent code bases and deployments to use only fully qualified DIDs in all communication. Unqualified DIDs received are matched by the receiving agent to the associated fully qualified DID.
```

Comment:
How will this work for legacy protocols? I'm thinking whether we should even update those. Rather provide a way for connections created using the legacy protocol to uograde their dids to a qualified format

Reply:
Not sure what you mean.  Do you mean specifically RFC 0160 Connection as the legacy protocol for establishing connections (should we update or drop that protocol), or something else?